src/Fossa/API/Types.hs

```diff
@@ -402,3 +405,14 @@ useApiOpts opts = case useURI serverURI of
 
 authHeader :: ApiKey -> Option 'Https
 authHeader key = header "Authorization" (encodeUtf8 ("Bearer " <> unApiKey key))
+
+data RevisionInfo = RevisionInfo
+  { revisionInfoLocator :: Text
+  , revisionInfoResolved :: Bool
+  }
+  deriving (Eq, Ord, Show)
+
+instance FromJSON RevisionInfo where
+  parseJSON = withObject "RevisionInfo" $ \obj ->
+    RevisionInfo <$> obj .: "locator"
+      <*> obj .: "resolved"
```

Comment:
[nit, optional] I think we generally put all these operators on newlines if we do any of them:
```suggestion
    RevisionInfo
      <$> obj .: "locator"
      <*> obj .: "resolved"
```